### DISCUSSION TITLE
Want to change label name in Filter criteria

### DISCUSSION THREAD

**Discussion by trusha82** [deejay-hub/timeline-lwc#244] [Q&A] [answered]
Hi Team,

I would like to change the label name in Filter criteria which appearing on Timeline during Filter selection(related object names). 

Currrently I have 'Completed Forms' filter name which is related object. But We have rename it to Case attachments in Salesforce now. I don't know how to change it in Timeline. Please refer to 

![Timeline_Filter](https://user-images.githubusercontent.com/52017492/232356869-104cbc0a-9d7d-4b6f-94e4-4f34875dc77c.PNG)
attach screenshot for more understanding.

I would like to change the name of 'Completed Forms' to Case Attachment'. Can you please advise me how to do it?

Thanks
Trusha

**Comment by deejay-hub** [comment, marked as answer]
Hi @trusha82. The filter panel gets the object label from the single and plural labels on the sObject.

<img width="1395" alt="image" src="https://user-images.githubusercontent.com/7021883/232924223-3f70dd4c-d32f-4006-bb01-494f31cd1a2e.png">

Then you'll need to refresh the browser.

What values do you have in the box highlighted above?

**Comment by deejay-hub** [comment]
@trusha82 Any update on what the values you have for single and plural labels are for your object?

**Reply by trusha82** [reply]
@deejay-hub Thank you so much for your response and apology for late reply. I managed to update the label name.
